src/test/java/com/serenitydojo/WhenWorkingWithStrings.java

```diff
@@ -0,0 +1,44 @@
+package com.serenitydojo;
+
+
+import org.junit.Test;
+
+public class WhenWorkingWithStrings {
+
+    //Exercise 1 - Converting to lower case
+@Test
```

Comment:
line intendation seems inconsistent. you want something that's automatically fixing that.  with intellij you can configure that java files are automatically formatted on saving https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html#reformat-on-save

Reply:
Done.